### PR TITLE
Unique ID gauth MFA

### PR DESCRIPTION
**Pull request by puntakana**
Hi,

To follow the PR #2904, I correct the bug in master branch.
I see @mmoayyed, you have created test case for that. Thanks.

All is-it ok ?

### CODE REVIEW

File: support/cas-server-support-gauth-core/src/main/java/org/apereo/cas/adaptors/gauth/repository/credentials/GoogleAuthenticatorAccount.java [hunk 1]
```diff
@@ -38,5 +38,6 @@ public GoogleAuthenticatorAccount(@JsonProperty("username") final String usernam
                                       @JsonProperty("validationCode") final int validationCode,
                                       @JsonProperty("scratchCodes") final List<Integer> scratchCodes) {
         super(username, secretKey, validationCode, scratchCodes);
+	this.setId(java.lang.System.currentTimeMillis());
```
Comment (mmoayyed) [inline support/cas-server-support-gauth-core/src/main/java/org/apereo/cas/adaptors/gauth/repository/credentials/GoogleAuthenticatorAccount.java]: Remove this please.

File: support/cas-server-support-otp-mfa/src/main/java/org/apereo/cas/otp/repository/credentials/OneTimeTokenAccount.java [hunk 2]
```diff
@@ -36,6 +36,7 @@
     private static final long serialVersionUID = -8289105320642735252L;
 
     @Id
+    @org.springframework.data.annotation.Id
     @GeneratedValue(strategy = GenerationType.AUTO)
     private long id = Integer.MAX_VALUE;
```
Comment (mmoayyed) [inline support/cas-server-support-otp-mfa/src/main/java/org/apereo/cas/otp/repository/credentials/OneTimeTokenAccount.java]: Remove the default value.

File: support/cas-server-support-otp-mfa/src/main/java/org/apereo/cas/otp/repository/credentials/OneTimeTokenAccount.java [hunk 3]
```diff
@@ -91,6 +92,10 @@ public String getUsername() {
         return username;
     }
 
+    public long getId() {
+		return id;
```
Comment (mmoayyed) [inline support/cas-server-support-otp-mfa/src/main/java/org/apereo/cas/otp/repository/credentials/OneTimeTokenAccount.java]: Fix indentation.

File: support/cas-server-support-gauth-core/src/main/java/org/apereo/cas/adaptors/gauth/repository/credentials/GoogleAuthenticatorAccount.java (hunk 1 above)
Comment (mmoayyed) [reply support/cas-server-support-gauth-core/src/main/java/org/apereo/cas/adaptors/gauth/repository/credentials/GoogleAuthenticatorAccount.java]: Actually I am wrong on this. Please leave this be.

**Comment by puntakana** [comment]
Hi @mmoayyed 
I see you merge my modification but I can't see this modification in the branch master (sorry, I don't know very well git & github)
So all is it OK for version 5.2 ?
Can I bring modification in branch 5.1 ?

Thanks

**Comment by mmoayyed** [comment]
You sure can, yes. Also include the tests and make sure they all pass. Thanks.

**Pull request merged**